src/renderers/widget.ts

```diff
@@ -381,6 +393,23 @@ function handleAnchor(anchor: HTMLAnchorElement, resolver: RenderMime.IResolver,
 
 
 /**
+ * Handle a link node.
+ */
+function handleLink(node: HTMLLinkElement, resolver: RenderMime.IResolver): Promise<void> {
```

Comment:
Do you think it would make sense to combine the handleLink and handleSource functions, which are essentially identical? handleAnchor is also similar, but I'm not sure exactly what the pathHandler is doing...